common/src/main/java/io/druid/collections/SerializablePair.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Licensed to Metamarkets Group Inc. (Metamarkets) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. Metamarkets licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.druid.collections;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.metamx.common.Pair;
+
+public class SerializablePair<T1, T2> extends Pair<T1, T2>
+{
+  public SerializablePair(@JsonProperty T1 lhs, @JsonProperty T2 rhs)
```

Comment:
I don't need it for this PR since I create the SerializablePair using object map in AggregatorFactory instead of using the object mapper, but it should be useful if this class is to be used in the future.